orca-core/src/main/java/com/netflix/spinnaker/orca/conditions/ConditionConfigurationProperties.java

```diff
@@ -29,7 +31,10 @@
   private boolean enabled = false;
   private Long backoffWaitMs = TimeUnit.MINUTES.toMillis(5);
   private Long waitTimeoutMs = TimeUnit.MINUTES.toMillis(120);
+  private List<String> clusters;
+  private List<String> conditions;
```

Comment:
What is the format of these conditions/clusters?

Reply:
A string for a condition doesn't seem very resilient. Or is it just a message to print?

Reply:
This is just a message to print. Aimed to be used for testing.